juniper_official/Kernel/krtasync.rule

```diff
@@ -213,6 +214,7 @@ healthbot {
             }
             field krt-io-task-name {
                 sensor krtiotable {
+                    where "krt-io-task-name == 'KRT IO task'";
```

Comment:
Double equal works only for interger hence use match operator =~

e.g. where "krt-io-task-name =~ 'KRT IO task'";

Reply:
it's working as expected with ==.
> select * from "external/check-krt-io-table_copy_3vpe40" order by desc limit 10
name: external/check-krt-io-table_copy_3vpe40
time                _instance_id _playbook_name krt-io-read-count krt-io-task-name krt-io-write-count tandIngestTimestamp tandTimeOffset
----                ------------ -------------- ----------------- ---------------- ------------------ ------------------- --------------
1592802789972650617 ["i1"]       test           11428             KRT IO task      26894              1592802789972411533 239.084µs
1592802728994468177 ["i1"]       test           11428             KRT IO task      26894              1592802728994275778 192.399µs
1592802666993114671 ["i1"]       test           11428             KRT IO task      26894              1592802666992883118 231.553µs
1592802605000207367 ["i1"]       test           11428             KRT IO task      26894